pfb-analysis/scripts/setup_database.sh

```diff
@@ -6,11 +6,18 @@ NB_POSTGRESQL_DB=pfb
 NB_POSTGRESQL_USER=gis
 NB_POSTGRESQL_PASSWORD=gis
 
+# Set configuration parameters
+su postgres bash -c psql <<EOF
+ALTER SYSTEM SET work_mem TO '4096MB';
```

Comment:
Just ran the new container, it does look like the server process restarts after running `setup_database.sh`.